lametro/templates/base.html

```diff
@@ -52,7 +52,10 @@
                                 <li><a class="dropdown-item dropdown-size" href="{% url 'committees' %}">Committees</a></li>
                                 <li><a class="dropdown-item dropdown-size" href="{% url 'lametro:contact' %}">Contact Us</a></li>
                                 <li><a class="dropdown-item dropdown-size" href="{% slugurl 'about' %}">About Us</a></li>
-                                <li><a class="dropdown-item dropdown-size" href="{% slugurl 'board-policies' %}">Board Policies</a></li>
+                                {% slugurl 'board-policies' as bp_url %}
+                                {% if bp_url != None %}
+                                    <li><a class="dropdown-item dropdown-size" href="{{ bp_url }}">Board Policies</a></li>
+                                {% endif %}
```

Comment:
It'll likely take some time for metro to set up this brand new page. So I wanted a way to not show the link until they create it.